Mathlib/Analysis/NormedSpace/PiTensorProduct/InjectiveSeminorm.lean

```diff
@@ -211,8 +211,7 @@ theorem injectiveSeminorm_le_projectiveSeminorm :
     existsi PUnit, inferInstance, inferInstance
     ext x
     simp only [Seminorm.zero_apply, Seminorm.comp_apply, coe_normSeminorm]
-    have heq : toDualContinuousMultilinearMap PUnit x = 0 := by ext _
```

Comment:
This `have` started specializing to `PUnit.{0}`, which broke the `rw` below. Inlining into the `rw` to force contextual unification seems to do the trick.